packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -490,6 +490,9 @@ export abstract class RestApiBase extends Resource implements IRestApi {
     ignore(deployment);
   }
 
+  /**
+   * @deprecated This method will be made internal. No replacement
```

Comment:
Doesn't it makes sense to create another @internal method called `_configureCloudWatchRole()` so this method won't get used?

Reply:
Done.